nuitka/Options.py

```diff
@@ -871,6 +872,27 @@ def commentArgs():
 Error, to compile a package, specify its directory but, not the '__init__.py'."""
             )
 
+    if isOnefileMode():
+        zstandard_supported_pythons = ("3.5", "3.6", "3.7", "3.8", "3.9", "3.10")
```

Comment:
This reaks of duplication, maybe use this via a local import (likely a cyclic dependency would otherwise trigger) from the `nuitka.freezer.onefile`, or make that call in `nuitka.MainControl`. I would like it for the warnings to come from the onefile logger, so a call of `getCompressorPython` will be all good. The found python is cached anyway.